<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EssaiGuiArticle</class>
 <widget class="QWidget" name="EssaiGuiArticle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>501</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>EssaiGuiArticle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../SeboApp.qrc">
    <normaloff>:/SeboApp/Resources/favicon.png</normaloff>:/SeboApp/Resources/favicon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitreAjoutArticle">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ajouter un article :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="laySaisie1">
     <item>
      <widget class="QLabel" name="lblLibelle">
       <property name="text">
        <string>Libellé :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSaisieLibelle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPV">
       <property name="text">
        <string>Prix vente (HT) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spPrixVente"/>
     </item>
     <item>
      <widget class="QLabel" name="lblGenre">
       <property name="text">
        <string>Genre :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbGenre"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="LaySaisie2">
     <item>
      <widget class="QLabel" name="lblFournisseur">
       <property name="text">
        <string>Fournisseur :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFournisseur"/>
     </item>
     <item>
      <widget class="QLabel" name="lblPA">
       <property name="text">
        <string>Prix d'achat :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spPrixFournisseur"/>
     </item>
     <item>
      <widget class="QCheckBox" name="ckReapprovisionnable">
       <property name="text">
        <string>Réapprovisionnable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layBtn1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnModifier">
       <property name="text">
        <string>Modifier</string>
       </property>
       <property name="icon">
        <iconset resource="../SeboApp.qrc">
         <normaloff>:/SeboApp/Resources/modifier.png</normaloff>:/SeboApp/Resources/modifier.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAjouter">
       <property name="text">
        <string>Ajouter</string>
       </property>
       <property name="icon">
        <iconset resource="../SeboApp.qrc">
         <normaloff>:/SeboApp/Resources/Ajouter.png</normaloff>:/SeboApp/Resources/Ajouter.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblTitreTable">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Liste des articles :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvArticles"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layBtn2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSupprimer">
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="icon">
        <iconset resource="../SeboApp.qrc">
         <normaloff>:/SeboApp/Resources/supprimer.png</normaloff>:/SeboApp/Resources/supprimer.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../SeboApp.qrc"/>
 </resources>
 <connections/>
</ui>
